project_stock/tests/common.py

```diff
@@ -88,6 +88,8 @@ def _create_task(self, products):
             self.env["project.task"].with_context(self._prepare_context_task(self))
         )
         task_form.name = "Test task"
+        # Save task to use default_get() correctlly in stock.moves
+        task_form.save()
```

Comment:
I don't get why you need this.

Reply:
Save is necessary to set `default_raw_material_task_id` (https://github.com/OCA/project/blob/13.0/project_stock/views/project_task_view.xml#L110) correctly and auto-create `procurement.group` record.

Reply:
That sound weird, and more if no stock.move is still created. Isn't this a bug on `Form`?

Reply:
Yes, it seems more like a bug in `Form()` but I think it's necessary for tests.